script.js

```diff
@@ -31,40 +41,114 @@ function onloadShow() {
     }
   });
 }
+//Call page on load function
+onloadShow();
+selectShow.addEventListener("change", (show) => {
+  if (show.target.value != "default") {
+    // results.style.display = "none";
+    let showValue = show.target.value;
+    console.log(showValue);
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
-  console.log(episodeList);
+    //Load shows
 
-  function onloadShow() {
-    allShows.forEach((show) => {
-      if (show.image) {
-        results.innerHTML += `
-                  <div class="show">
-                         <div class="show-preview">
-                         <h2>${show.name}</h2>
-                         <img src="${show.image.medium}">
-                         <a href="#">View all Episodes <i class="fas fa-chevron-right"></i></a>
-                       </div>
+    fetch(`https://api.tvmaze.com/shows/${showValue}/episodes`)
+      .then((res) => res.json())
+      .then((episodes) => {
+        //Episodes function
+        function makePageForEpisodes() {
+          results.innerHTML = "";
+          selectInput.innerHTML = "";
+          episodes
+            .filter(
+              (episode) =>
+                episode.name.toLowerCase().includes(term.toLowerCase()) ||
+                episode.summary.toLowerCase().includes(term.toLowerCase())
+            )
+            .forEach((episode) => {
+              if (episode.image) {
+                results.innerHTML += `<div>
+   <h2>${episode.name} - S${zeroPadded(episode.season)}E${zeroPadded(
+                  episode.number
```

Comment:
Good work Georgina, I like that you made a function to add the episodes and the seasons numbers .

Reply:
Thanks Halla-S